clean_helpers/clean_pseudocrawl.py

```diff
@@ -0,0 +1,11 @@
+def remove_lines_with_curly_brackets(examples):
+    """Removes lines containing a '{' from the texts."""
+    fixed_texts = []
+    for text in examples["text"]:
+        fixed_lines = []
+        for line in text.split("\n"):
+            if "{" not in line:
+                fixed_lines.append(line)
+        fixed_texts.append("\n".join(fixed_lines))
+    examples["text"] = fixed_texts
+    return examples
```

Comment:
```suggestion
    return {
        **examples,
        "text": ["\n".join([line for line in text.split("\n") if "{" not in line]) for text in example["text"]]
    }
```

Are we not matching `}` then?

Reply:
Good point. I add a `}` case to be safe. On a quick test it on a few dataset it added a handful of fixes.